xml/schema/CRM/Sqltasks/SqltasksActionTemplates.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
```

Comment:
The file name uses plural here; the convention in Civi is to use singular, plus `civix` and other parts of Civi expect schema files and their corresponding classes to have matching names.